powerauth-push-server/src/main/java/com/wultra/push/errorhandling/DefaultExceptionHandler.java

```diff
@@ -147,14 +147,8 @@ public ErrorResponse handleAppNotFoundException(AppNotFoundException e) {
     @ExceptionHandler(MethodArgumentNotValidException.class)
     @ResponseStatus(HttpStatus.BAD_REQUEST)
     public @ResponseBody ErrorResponse handleMethodArgumentNotValidException(MethodArgumentNotValidException e) {
-        logger.warn("Error occurred when calling an API: {}", e.getMessage());
-        logger.debug("Exception detail: ", e);
-        final ExtendedError error = new ExtendedError("ERROR_REQUEST", "Invalid method parameter value");
-        for (FieldError fieldError : e.getBindingResult().getFieldErrors()) {
-            error.getViolations().add(
-                    new Violation(fieldError.getField(), fieldError.getRejectedValue(), fieldError.getDefaultMessage())
-            );
-        }
-        return new ErrorResponse(error);
+        logger.error(e.getMessage(), e);
```

Comment:
🤔 Was this change really needed? I just wanted to know whether introducing bean validation would not break API behavior, but it seems that the exception handler is already prepared for that. What was your motivation to change it?